testutils/clustermanager/e2e-tests/boskos/boskos_test.go

```diff
@@ -71,49 +70,53 @@ func TestAcquireGKEProject(t *testing.T) {
 	defer func() {
 		common.GetOSEnv = oldGetOSEnv
 	}()
-	for _, data := range datas {
-		setup()
-		ts := fakeServer(func(w http.ResponseWriter, r *http.Request) {
-			if data.serverErr {
-				http.Error(w, "", http.StatusBadRequest)
-			} else {
-				// RequestURI for acquire contains a random hash, doing
-				// substring matching instead
-				for _, s := range []string{"/acquire?", "owner=" + data.expHost, "state=free", "dest=busy", "type=gke-project"} {
-					if !strings.Contains(r.RequestURI, s) {
-						t.Fatalf("request URI doesn't match: want: contains '%s', got: '%s'", s, r.RequestURI)
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			setup()
+			ts := fakeServer(func(w http.ResponseWriter, r *http.Request) {
+				if tt.serverErr {
+					http.Error(w, "", http.StatusBadRequest)
```

Comment:
I'd suggest a non-empty string here that can be verified to make sure error propagation etc works as desired.